File: we_9_2_a_levenberg_marquardt_accept_reject_decision.m
% Integrated Navigation Systems — Chapter 9: Factor Graphs, Nonlinear Optimization, and AI-Aided Estimation
% Worked Example 9.2 — A Levenberg–Marquardt accept/reject decision
% Companion code (educational reference implementation; see the book for context).

f0=7.0; f1=0.375;
if f1<f0, disp('accept, decrease lambda'); else disp('reject, increase lambda'); end

File: we_9_2_a_levenberg_marquardt_accept_reject_decision.py
# Integrated Navigation Systems — Chapter 9: Factor Graphs, Nonlinear Optimization, and AI-Aided Estimation
# Worked Example 9.2 — A Levenberg–Marquardt accept/reject decision
# Companion code (educational reference implementation; see the book for context).

f0,f1=7.0,0.375
print('accept, decrease lambda' if f1<f0 else 'reject, increase lambda')
